File: .utils/uptime/uptime.pl
#!/usr/local/bin/perl

use Unix::Uptime;
use POSIX;

my $uptime = Unix::Uptime->uptime();
my $days = floor($uptime / 86400);
$uptime = $uptime - ($days * 86400);
my $hours = floor($uptime / 3600) % 24;
$uptime = $uptime - ($hours * 3600);
my $minutes = floor($uptime / 60) % 60;
$uptime = $uptime - ($minutes * 60);
my $up = "$days:$hours:$minutes";

print "$up";

File: .utils/uptime/uptime.js
#!/usr/local/bin/node

let uptime = require('os').uptime()
const days = Math.floor(uptime / 86400)
uptime -= days * 86400
const hours = Math.floor(uptime / 3600) % 24
uptime -= hours * 3600
const minutes = Math.floor(uptime / 60) % 60
uptime -= minutes * 60
console.log(`${days ? `${days}:` : ''}${hours}:${minutes}`)
